### PR TITLE
zh-CN: translate memory-management

### PR DESCRIPTION
**Pull request by mgeisler**
Part of #324.

### CODE REVIEW

File: po/zh-CN.po [hunk 1]
```diff
@@ -3834,40 +3834,44 @@ msgstr ""
...
 msgid "Rust offers a new mix:"
-msgstr ""
+msgstr "Rust 提供了一个全新的组合："
 
 #: src/memory-management.md:10
 msgid ""
 "> Full control *and* safety via compile time enforcement of correct memory\n"
 "> management."
 msgstr ""
+"> 通过编译时强制执行正确的内存\n"
```
Comment (wnghl) [inline po/zh-CN.po:3861]:
```suggestion
"> 通过编译时强制执行正确的内存"
```

File: po/zh-CN.po [hunk 2]
```diff
@@ -3834,40 +3834,44 @@ msgstr ""
...
+">管理来实现完全控制与安全。"
 
 #: src/memory-management.md:13
 msgid "It does this with an explicit ownership concept."
-msgstr ""
+msgstr "它通过一个明确的所有权概念来实现此目的。"
```
Comment (wnghl) [inline po/zh-CN.po:3866]:
```suggestion
msgstr "它通过一个明确的所有权（ownership）概念来实现此目的。"
```

File: po/zh-CN.po [hunk 3]
```diff
@@ -3882,17 +3886,29 @@ msgid ""
 "  * Slightly slower than the stack: some book-keeping needed.\n"
 "  * No guarantee of memory locality."
 msgstr ""
+"* 栈：局部变量的连续内存区域。\n"
+"  * 值在编译时具有已知的固定大小。\n"
+"  * 速度极快：只需移动一个栈指针。\n"
+"  * 易于管理：遵循函数调用规则。\n"
+"  * 实现内存局部性的出色方案。\n"
```
Comment (wnghl) [inline po/zh-CN.po:3893]:
```suggestion
"  * 优秀的内存局部性。\n"
```

File: po/zh-CN.po [hunk 4]
```diff
@@ -4037,11 +4117,15 @@ msgid ""
 "* At the end of `say_hello`, the `std::unique_ptr` destructor will run.\n"
 "* The destructor frees the `Person` object it points to."
 msgstr ""
+"* `std::unique_ptr` 对象在栈上分配内存，并指向\n"
```
Comment (wnghl) [inline po/zh-CN.po:4120]:
```suggestion
"* `std::unique_ptr` 对象在栈上分配内存，并指向"
```

**Review by superwhd — APPROVED** [review]
LGTM 👍

**Comment by mgeisler** [comment]
Thanks @wnghl and @superwhd for the reviews!

**Pull request merged**